config/git/src/test/kotlin/GitConfigFileProviderFactoryTest.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.config.git
+
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.core.test.TestCase
+import io.kotest.core.test.TestResult
+import io.kotest.matchers.shouldBe
+
+import io.mockk.every
+import io.mockk.mockkConstructor
+import io.mockk.unmockkAll
+
+import java.io.ByteArrayInputStream
+
+import org.eclipse.apoapsis.ortserver.config.ConfigManager
+import org.eclipse.apoapsis.ortserver.config.ConfigSecretProviderFactoryForTesting
+import org.eclipse.apoapsis.ortserver.config.Context
+import org.eclipse.apoapsis.ortserver.config.Path
+
+private const val CONTENT = "Hello world"
+
+class GitConfigFileProviderFactoryTest : WordSpec() {
+    override suspend fun afterEach(testCase: TestCase, result: TestResult) {
+        unmockkAll()
+    }
+
+    init {
```

Comment:
Please apply the same "compactifications" as done [here](https://github.com/eclipse-apoapsis/ort-server/pull/412).